com.autodesk.Eagle.yml

```
@@ -0,0 +1,53 @@
+app-id: com.autodesk.Eagle
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: eagle
+build-options:
+  no-debuginfo: true
+finish-args:
+  - --socket=x11
+  - --device=dri
+  - --share=network
+  - --share=ipc
+  - --filesystem=home
+modules:
+  - name: eagle
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 com.autodesk.Eagle.appdata.xml /app/share/appdata/${FLATPAK_ID}.appdata.xml
+      - install -Dm644 com.autodesk.Eagle.desktop /app/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 eagle-logo.png /app/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png
+      - install -Dm644 mime.xml /app/share/mime/packages/${FLATPAK_ID}.xml
+      - install -Dm755 eagle.sh /app/bin/eagle
+      - install -Dm755 apply_extra.sh /app/bin/apply_extra
+    sources:
+      - type: extra-data
+        url: https://eagle-updates.circuits.io/downloads/9_6_2/Autodesk_EAGLE_9.6.2_English_Linux_64bit.tar.gz
+        filename: eagle.tar.gz
+        sha256: 0086b3fddcd9eaeead0f30f814b868a9e0ef75fbdf6185ec3448d0a9edf35ea3
+        size: 171780187
+        only-arches: [x86_64]
+        x-checker-data:
+          type: rotating-url
+          url: https://www.autodesk.com/eagle-download-lin
+      - type: script
+        dest-filename: apply_extra.sh
+        commands:
+          - tar xzf eagle.tar.gz
+          - mv eagle-*/* .
+          # Remove provided X libraries. They cause issues with current mesa and the system libraries seem to work fine.
+          - rm lib/libx* lib/libX*
+          - install -Dm644 doc/eagle.1 export/share/man/man1/eagle.1
```

```suggestion
          - install -Dm644 doc/eagle.1 export/share/man/man1/eagle.1
```

Nobody will see the manpage.